### PR TITLE
Start Video without Audio Focus

### PR DESCRIPTION
**Pull request by S3basuchian**
I have added two options to `IFramePlayerOptions `:

- `autoplay`
- `mute`

If both of these values are initialized with 1, the video will autoplay in a muted mode without gaining Audio Focus (meaning Spotify or other background playbacks will continue). As soon as the player is unmuted with the `unMute()` function, it will acquire audio focus again.
I think this should fix the issues #691 and #397

### CODE REVIEW

**Comment by PierfrancescoSoffritti** [comment]
Hey, thanks for the PR! can you please send it to the dev branch instead of master?

I will try to review it over the weekend :)

**Comment by S3basuchian** [comment]
Sure thing :)

File: README.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ A lengthier explanation to why you may want to consider using an alternative to
 
 A list of published apps that are using this library: ([let me know](https://github.com/PierfrancescoSoffritti/android-youtube-player/issues) if you want to add your app to this list)
 
+- [Genius](https://play.google.com/store/apps/details?id=com.genius.android)
```
Comment (PierfrancescoSoffritti) [inline README.md:32]: mm not sure why this is here, i might have messed up when i updated the readme 😬  anyway, to keep this change clean, can you remove this line?

File: core/src/main/java/com/pierfrancescosoffritti/androidyoutubeplayer/core/player/options/IFramePlayerOptions.kt [hunk 2]
```diff
@@ -66,6 +68,24 @@ class IFramePlayerOptions private constructor(private val playerOptions: JSONObj
...
+         * Controls if the video is played automatically.
+         * @param autoplay if set to 1: the player will start automatically. If set to 0: the player will not start automatically
+         */
+        fun autoplay(controls: Int): Builder {
+            addInt(AUTO_PLAY, controls)
+            return this
+        }
+
+        /**
+         * Controls if the video is muted.
```
Comment (PierfrancescoSoffritti) [inline core/src/main/java/com/pierfrancescosoffritti/androidyoutubeplayer/core/player/options/IFramePlayerOptions.kt:81]: Change to something like: "Controls if the player will be initialized mute or not."

File: core/src/main/java/com/pierfrancescosoffritti/androidyoutubeplayer/core/player/options/IFramePlayerOptions.kt [hunk 3]
```diff
@@ -66,6 +68,24 @@ class IFramePlayerOptions private constructor(private val playerOptions: JSONObj
             return this
         }
 
+        /**
+         * Controls if the video is played automatically.
```
Comment (PierfrancescoSoffritti) [inline core/src/main/java/com/pierfrancescosoffritti/androidyoutubeplayer/core/player/options/IFramePlayerOptions.kt:72]: Change to something like: "Controls if the video is played automatically after the player is initialized"

**Comment by S3basuchian** [comment]
I have removed the readme changes and changed the wording of the method descriptions 👍🏻

**Review by PierfrancescoSoffritti — APPROVED** [review]
Looks good to me!

**Comment by PierfrancescoSoffritti** [comment]
Thanks for the help :)

**Pull request merged**